### PR TITLE
More Find Methods

### PR DESCRIPTION
**Pull request by smizell**
This PR introduces two new methods for finding elements. They are mostly helper methods that reuse the functionality of `find` and allow for finding all elements given an element name and all elements given a class.

### CODE REVIEW

**Comment by smizell** [comment]
cc @fosrias @danielgtaylor if either of you all can get to it today

File: README.md [hunk 1]
```diff
@@ -292,6 +292,14 @@ var numbers = arrayElement.find(function(el) {
 }).toValue(); // [1, 2, 3]
 ```
 
+##### findByClass
+
+The `findByClass` method traverses the entire descendent element tree and returns `ArrayElement` of all elements that match the given element name.
```
Comment (fosrias) [inline README.md]: returns an `ArrayElement`... Ditto below

File: test/primitives-test.js [hunk 2]
```diff
@@ -599,6 +602,38 @@ describe('Minim Primitives', function() {
...
 
+      describe('#findByElement', function() {
+        var items;
+
+        before(function() {
+          items = doc.findByElement('number');
+        });
+
+        it('returns the correct number of items', function() {
+          expect(items.length).to.equal(1);
```
Comment (fosrias) [inline test/primitives-test.js]: can you say expect(items).to.have(1).item or something like that in mocha?

File: test/primitives-test.js [hunk 3]
```diff
@@ -599,6 +602,38 @@ describe('Minim Primitives', function() {
...
+
+      describe('#findByClass', function() {
+        var items;
+
+        before(function() {
+          items = doc.findByClass('test-class');
+        });
+
+        it('returns the correct number of items', function() {
+          expect(items.length).to.equal(1);
```
Comment (fosrias) [inline test/primitives-test.js]: ditto per above

**Comment by smizell** [comment]
@fosrias Not sure if you were done, but check latest commit that addresses comments.

**Comment by danielgtaylor** [comment]
Looks good to me :+1:

**Pull request merged**